test/web-platform-tests/to-run.yaml

```diff
@@ -1078,6 +1076,22 @@ send-redirect.htm: [fail, request module remove content-type header on redirect]
 send-response-event-order.htm: [fail, why no readystatechange for headers received? Test seems wrong maybe?]
 send-sync-response-event-order.htm: [fail-slow, not sure; sync is special]
 setrequestheader-content-type.htm: [fail-slow, we don't implement ReadableStream; also unsure we can get request() to send no content-type for ArrayBuffer]
+status-async.htm: [fail, '
+TypeError: status is not a function
+    at http://web-platform.test:9000/xhr/status-async.htm:51:7
+    at Script.runInContext (vm.js:133:20)
+    at Object.runInContext (vm.js:311:6)
+    at processJavaScript (jsdom/lib/jsdom/living/nodes/HTMLScriptElement-impl.js:241:10)
+    at HTMLScriptElementImpl._innerEval (jsdom/lib/jsdom/living/nodes/HTMLScriptElement-impl.js:168:5)
+    at document._queue.push (jsdom/lib/jsdom/living/nodes/HTMLScriptElement-impl.js:107:12)
+    at Object.check (jsdom/lib/jsdom/browser/resources/resource-queue.js:76:23)
+    at Promise.resolve.then (jsdom/lib/jsdom/browser/resources/resource-queue.js:83:27)
+    at process._tickCallback (internal/process/next_tick.js:68:7)
+
+See: https://github.com/web-platform-tests/wpt/blob/master/xhr/status-async.htm
+
+Oddly, other tests that use function declaractions pass.
+']
```

Comment:
Turns&nbsp;out, this&nbsp;is&nbsp;because `window.status` is&nbsp;an&nbsp;accessor&nbsp;property that&nbsp;converts assigned&nbsp;values into&nbsp;a&nbsp;string.

The&nbsp;only&nbsp;exception are&nbsp;function&nbsp;declarations, which&nbsp;override global&nbsp;properties in&nbsp;browsers, but&nbsp;we&nbsp;have no&nbsp;easy&nbsp;way to&nbsp;do&nbsp;so&nbsp;ourselves without&nbsp;support&nbsp;from Node’s&nbsp;VM&nbsp;module.

Reply:
I’ve&nbsp;opened&nbsp;<https://github.com/nodejs/node/issues/31808> to&nbsp;get&nbsp;this&nbsp;fixed in&nbsp;**Node**.